NOTICE.txt

```diff
@@ -1,2 +1,30 @@
+
 ARLAS-server
-Copyright 2017 Gisaïa
\ No newline at end of file
+Copyright 2017 and onwards Gisaïa
```

Comment:
2017 - 2018 ?

Reply:
`2017 and onwards` requires less maintenance each year ;)